app/assets/javascripts/google-analytics-events.js

```diff
@@ -0,0 +1,67 @@
+/*
+ * Tracks custom events with Google Analytics.
+ * This application uses the google-analytics-rails gem to track general usage
+ * data. This file defines additional user interations to be tracked with Google
+ * Analtyics.
+ * See https://developers.google.com/analytics/devguides/collection/analyticsjs/events
+ */
+
+$(function(){
```

Comment:
This syntax is shorthand for `$(document).ready(function() { ... });` so the `$(document).ready()` on line 12 can be removed.